pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix

```
@@ -196,7 +196,15 @@ in rec {
     # Path -> PathGlob -> [Path]
     expandGlob = base: glob: expandGlobList base (splitGlob glob);
 
-    packagePaths = lib.concatMap (expandGlob src) packageGlobs;
+    validPackage = path:
+      let
+        packageJSON = path+"/package.json";
```

nit: I personally prefer this expression:


```suggestion
        packageJSON = "${path}/package.json";
```